tests/test_module_list.py

```diff
@@ -33,6 +34,24 @@ def test_public_api(self):
             mod.append(code_folder)
         self.assertEqual(sorted(monai.__all__), sorted(mod))
 
+    def test_transform_api(self):
+        """monai subclasses of MapTransforms must have alias names ending with 'd', 'D', 'Dict'"""
+        to_exclude = {"MapTransform"}  # except for these transforms
+        xforms = {
+            name: obj
+            for name, obj in monai.transforms.__dict__.items()
+            if inspect.isclass(obj) and issubclass(obj, monai.transforms.MapTransform)
+        }
+        names = sorted(x for x in xforms if x not in to_exclude)
+        remained = set(names)
+        for n in names:
+            if not n.endswith("d"):
+                continue
+            basename = n[:-1]  # Transformd basename is Transform
+            for postfix in ("D", "d", "Dict"):
+                remained.remove(f"{basename}{postfix}")
```

Comment:
Will this line raise exception if `{basename}{postfix}` doesn't exist?
If not, we need to check the `{basename}{postfix}` must in `remained`.

Thanks.